governance-server/index.js

```diff
@@ -9,6 +9,8 @@ logic = null;
 
 const GIT_REV = null;
 
+BigInt.prototype.toJSON = function () { return Number(this) }
```

Comment:
needed as we can't `JSON.stringify` with bigint, currently all data is optionally cached using strings. This will break for larger numbers but hasn't been a problem in the past. A refactor wouldn't be trivial